src/Markdig/Helpers/StringSlice.cs

```diff
@@ -7,6 +7,50 @@
 
 namespace Markdig.Helpers
 {
+    /// <summary>
+    /// Wrap newline so we have type-safety and static accessibility
+    /// </summary>
+    public enum Newline
```

Comment:
Rename to `NewLine` and move it to its own separated file together with the NewLine extensions below.

Reply:
As Wikipedia and Wiktionary suggest newline started as a compound word, but over time became a single word. I have moved Newline to a separate file. I strongly suggest to use Newline, but if you insist, I'll gladly make it NewLine.
https://en.wikipedia.org/wiki/Newline
https://en.wiktionary.org/wiki/newline

Reply:
The .NET framework is using names like `Environment.NewLine`, so I would like that this to be consistent.

Reply:
That's fair, I'll change it to NewLine on the class and member names.